### PR TITLE
feat: Multi wacz json Front End Support

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
Resolves https://github.com/webrecorder/replayweb.page/issues/208

Add Display values for:

- File size of combined WACZ
- List of all WACZ files (with downloadable links)
- Description and Title from Collection file
- Description and Title on pages list as well

Tested on https://dev.browsertrix.cloud/orgs/44bf96c1-6007-4521-9313-4e927cd9862a/collections/view/c2d39b06-48d0-4454-84d1-444aa2d4b91e

*Need to make sure it doesn't go funky with non-MultiWACZ files*

image

image

You'll want to review the commits individually. The first one is the changes made, the second is formatting / linting

### CODE REVIEW

**Comment by Chickensoupwithrice** [comment]
Rebased on main after we formatted it all, and made sure to run `yarn format` after my changes too.

**Comment by ikreymer** [comment]
nice! Some other things that would be good to add:
- Using the same collection title on the breadcrumbs and index page so it doesn't say 'replay.json' there.
- On the info tab, maybe instead of showing filename: replay.json we instead show:
```
Collection:
<Collection Name>

<Desc> (if any)

Files:
...
```

Maybe also add numbered list to files?

We should also add a way to download the entire collection, but that'll require some wabac.js changes, and can be a later pass. Can also update the API there to include a type like 'WACZ' or 'Multi WACZ Collection' but probably best done with wabac.js changes.

**Comment by ikreymer** [comment]
> I've added the collection name to the breadcrumb, which was a small fix.
> 
> However for the second change I think we're better served by making the changes in wabac.js first and then using that to determine how the info tab should be displayed? Sprinkling in a bunch of conditionals based on whether or not `collInfo.name` exists seems to me the wrong approach? Doubly so if we're going to be redoing this page soon. Let me know what you think, happy to be corrected / do this anyway :)

Hm, is it more work to get it to the coll info listing? Mostly just for consistency reasons, so its the same thing listed there, otherwise all our public collection show up as 'replay.json'.

The title page isn't being redone yet, but probably changes to wabac.js api will make it easier. But perhaps that can just be a separate iteration along with downloading.

**Comment by Chickensoupwithrice** [comment]
Ohhh, sorry, I think I understand now. That actually should be pretty easy, I'll do it now :) 

I think wabac.js changes would make it slightly easier, but like you said we can do that in a separate iteration

**Comment by Chickensoupwithrice** [comment]
Example: 

image

I think this is what you meant? Also a very small change

**Comment by Chickensoupwithrice** [comment]
Now with numbered list:
image

And home page showing collection name if it exists:
image

File: src/coll-index.js [hunk 1]
```diff
@@ -513,7 +513,7 @@ class CollInfo extends LitElement {
             ${coll.desc}
           </div>`
         : html``}
-      /* Only show description if it exists, otherwise show filename? */
+      <!--  Only show description if it exists, otherwise show filename -->
```
Comment (ikreymer) [inline src/coll-index.js]: Minor tweak: if we have description show it. If we have `coll.resources`, then don't show the filename as it'll just be `replay.json` and we're already showing that below and its

Comment (ikreymer) [reply src/coll-index.js]: Some maybe add show filename as else condition for coll.resources

**Comment by ikreymer** [comment]
Just one final comment for if there's no description, otherwise ready to go!

**Pull request merged**